src/main/kotlin/com/bada/app/models/RegisterCustomer.kt

```diff
@@ -0,0 +1,35 @@
+package com.bada.app.models
+
+import com.bada.app.auth.CustomerUserDetails
+import com.bada.app.repos.CompanyRepository
+import com.bada.app.repos.CountryRepository
+import javax.persistence.*
+
+class RegisterCustomer(
+    var username: String,
+    var password: String,
+    var email: String,
+    var firstName: String,
+    var lastName: String,
+    var nip: String,
+    var phoneNumber: String,
+    var addressLine1: String,
+    var addressLine2: String,
+    var zipcode: String?,
+    var city: String?,
+
+    ) {
+    constructor() : this(
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+    )
+}
```

Comment:
Are these empty strings necessary? If so, move them as the default values of the default constructor.